nilearn/plotting/surface/surf_plotting.py

```diff
@@ -623,30 +636,31 @@ def plot_surf_stat_map(
         symmetric_cbar=symmetric_cbar,
     )
 
-    fig = backend._plot_surf(
+    fig = plot_surf(
         surf_mesh,
         surf_map=loaded_stat_map,
         bg_map=bg_map,
         hemi=hemi,
         view=view,
-        avg_method=avg_method,
-        threshold=threshold,
+        engine=engine,
         cmap=cmap,
-        symmetric_cmap=True,
+        symmetric_cmap=symmetric_cmap,
         colorbar=colorbar,
-        cbar_tick_format=cbar_tick_format,
+        avg_method=avg_method,
+        threshold=threshold,
         alpha=alpha,
         bg_on_data=bg_on_data,
         darkness=darkness,
-        vmax=vmax,
         vmin=vmin,
+        vmax=vmax,
+        cbar_vmin=cbar_vmin,
+        cbar_vmax=cbar_vmax,
+        cbar_tick_format=cbar_tick_format,
```

Comment:
Here just reordering of params, to make it easy to compare with `plot_surf` signature.